examples/example_plugin/example_plugin/jobs.py

```diff
@@ -56,7 +59,7 @@ class Meta:
             *This is italicized*
         """
 
-    def run(self):
+    def run(self, some_json_data):
```

Comment:
Just checking - is `some_json_data` as presented to the `run()` method a JSON string, or a Python dict derived from the provided JSON? It would be good to document and test in either case.

Reply:
Added notes to docs